cli/lib/appdata.ts

```diff
@@ -92,9 +102,24 @@ export async function saveAppdata( userData: UserData ): Promise< void > {
 		const fileContent = JSON.stringify( userData, null, 2 ) + '\n';
 
 		fs.writeFileSync( appDataPath, fileContent, 'utf8' );
+	} catch ( error ) {
+		throw new LoggerError( __( 'Failed to save appdata file' ), error );
+	}
+}
+
+export async function getAuthToken(): Promise< NonNullable< UserData[ 'authToken' ] > > {
+	try {
+		const { authToken } = await readAppdata();
+
+		if ( ! authToken?.accessToken ) {
+			throw new Error( 'Authentication required' );
```

Comment:
Nit: Don't we need a translated string here?

Reply:
We don't actually. We rely on the `LoggerError` thrown on L120 for the translated string.